### PR TITLE
fix: inline config interfaces

### PR DESCRIPTION
**Pull request by pjungermann**
Signed-off-by: Patrick Jungermann <Patrick.Jungermann@gmail.com>

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by pjungermann** [comment]
@Rugvip this change is related to https://github.com/backstage/backstage/pull/11345#discussion_r872533107

File: .changeset/beige-deers-remember.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-cost-insights': patch
+---
+
+Add missing `export`
```
Comment (Rugvip) [inline .changeset/beige-deers-remember.md:5]:
```suggestion
Add missing `export` in configuration schema.
```

**Review by Rugvip — APPROVED** [review]
Nice, thank you! 😁

File: .changeset/beige-deers-remember.md (hunk 1 above)
Comment (pjungermann) [reply .changeset/beige-deers-remember.md]: fixed.

**Pull request merged**